docs/pbc/all/data-exchange/202311.0/tutorials-and-howtos/how-to-send-request-in-data-exchange-api.md

```diff
@@ -146,6 +147,70 @@ If you send a request to an endpoint that's not configured in `spy_dynamic_entit
 ]
 ```
 
+### Retrieve a collection of entities together with relations
+
+To retrieve a collection of countries with relations, you need to send the `GET https://glue.mysprykershop.com/dynamic-entity/countries?include=countryTaxRates` request.
+`include` parameter allows to define a list of relations that we want to retrieve by the request together with the main entity,
+it has the following format `{mainEntity}?include={childRelationOfTheMainEntity.childRelationOfTheChildRelation},{secondRelationChain},{thirdRelationChain}`, where:
+- Include parameter can consist from one or several relation chains with `,` delimiter.
+- Each relation chain represents a list of relation names (See spy_dynamic_entity_configuration_relation.name) separated with `.` delimiter`.
+- Each relation in the relation chain is a connection between two `spy_dynamic_entity_configuration`, see `spy_dynamic_entity_configuration_relation` table structure.
+
+```bash
+GET /dynamic-entity/countries?include=countryTaxRates HTTP/1.1
+Host: glue-backend.mysprykershop.com
+Content-Type: application/json
+Accept: application/json
+Authorization: Bearer {your_token}
+```
+
+Response sample:
+
+```json
+{
+  "data": [
+    "...",
+    {
+      "id_country": 14,
+      "iso2_code": "AT",
+      "iso3_code": "AUT",
+      "name": "Austria",
+      "postal_code_mandatory": true,
+      "postal_code_regex": "\\d{4}",
+      "countryTaxRates": [
+        {
+          "id_tax_rate": 1,
+          "fk_country": 14,
+          "name": "Austria Standard",
+          "rate": "20.00"
+        },
+        {
+          "id_tax_rate": 21,
+          "fk_country": 14,
+          "name": "Austria Reduced1",
+          "rate": "13.00"
+        },
+        {
+          "id_tax_rate": 22,
+          "fk_country": 14,
+          "name": "Austria Reduced2",
+          "rate": "10.00"
+        }
+      ]
+    },
+    "..."
+  ]
+}
+```
+
+The response contains all the columns from the `spy_country` table and included `spy_tax_rate` table that are configured in `spy_dynamic_entity_definition.definition`. Each column is represented using the `fieldVisibleName` as the key, providing a comprehensive view of the table's data in the API response.
+
+{% info_block infoBox %}
+
+Currently, filters do not work with `relations` and only used to filter the root entity results.
```

Comment:
Please specify version as an example. We can forget about this line and we should at least give a hint when it was not working